block_manipulation_service/src/main/java/com/stnetix/ariaddna/blockmanipulation/LocalService.java

```diff
@@ -0,0 +1,10 @@
+package com.stnetix.ariaddna.blockmanipulation;
+
+import java.io.File;
+
+/**
+ * Created by LugovoyAV on 16.02.2018.
+ */
+public interface LocalService {
```

Comment:
Please create module/or update project module for LocalService.
(project module may contain only this interface now)